### ISSUE TITLE
Deprecated warnings are appearing.

### ISSUE THREAD

**Issue by kaito0228** [navarasu/serverless-ruby-layer#29]
**Describe the warning**
Deprecated warnings are appearing.
I need to set the config as shown in the message, and then run a normal bundle install.
`bundle config set without 'test development'`

**Error Stack Trace**
```
Serverless: [DEPRECATED] The `--without` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set without 'test development'`, and stop using this flag
```

**Example Config files**
```
plugins:
  - serverless-ruby-layer
```

**Desktop (please complete the following information):**
 - OS: Mac OS
 - NodeVersion 14.14.0
 - Ruby version 2.7.0
 - Bundler version 2.1.4

**Additional context**

**Comment by navarasu** [comment]
Thanks, Ryo. It will be addressed in this weekend release

**Comment by navarasu** [comment]
@kaito0228 This issue is addressed in release 1.4.0. Please update the npm version and verify.
```
npm i serverless-ruby-layer
```

**Comment by kaito0228** [comment]
Thanks!
